app/views/shelters/show.html.erb

```diff
@@ -3,6 +3,12 @@
   <p><%= @shelter.address %></p>
   <p><%= @shelter.city %>, <%= @shelter.state %> <%= @shelter.zip %></p>
   <br>
+
+  <section>
```

Comment:
This will be where the shelter's reviews go. I just got waylaid with other issues and didn't get around to fully implementing it quite yet. Also, I arbitrarily chose a section tag. I'm not sure if it would be better in a different place on the page with a different tag.

Reply:
This should be a good place to start.  A `section` tag should work fine for our purposes, and we could give it an `id` of "review" or something along those lines.